src/views/product-tutorials-view/server.ts

```diff
@@ -8,9 +9,13 @@ import {
 import { stripUndefinedProperties } from 'lib/strip-undefined-props'
 import { filterCollections } from './helpers'
 
+// Some of the product data is coming from the API client on this view
+type ProductTutorialsPageProduct = ClientProduct &
```

Comment:
Some context on the switch here: the base `LearnProductData` is being used as what is passed in to the `getStaticProps` calls and that shouldn't yet include the API client data. 

And there is only one view currently that uses this combination of types so keeping that close to this file as opposed to the global type.